arrow-udf/Cargo.toml

```diff
@@ -10,6 +10,7 @@ license = "Apache-2.0"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [features]
+default = ["global_registry"]
```

Comment:
We don't enable the global registry by default as it might cause errors when used for building UDF modules on Wasm.

```suggestion
```

Reply:
My bad. removed.